ipm_library/setup.py

```diff
@@ -0,0 +1,25 @@
+from setuptools import setup
+
+package_name = 'ipm_library'
+
+setup(
+    name=package_name,
+    version='0.0.0',
+    packages=[package_name],
+    data_files=[
+        ('share/ament_index/resource_index/packages',
+            ['resource/' + package_name]),
+        ('share/' + package_name, ['package.xml']),
+    ],
+    install_requires=['setuptools'],
+    zip_safe=True,
+    maintainer='florian',
+    maintainer_email='git@flova.de',
+    description='Inverse Perspective Mapping ROS2 Library',
+    license='MIT',
```

Comment:
```suggestion
    license='Apache License, Version 2.0',
```

Reply:
Up here (https://github.com/ros-sports/ipm/pull/1#discussion_r826627489), you suggested `Apache License 2.0` without the `, Version` part. I'm not quite sure, what's correct, but I guess it should match?

Reply:
I copied this across from [ros2cli's setup.py](https://github.com/ros2/ros2cli/blob/478a430dfb46403d6b156a6b1dc8f531e9a4b3d8/ros2pkg/setup.py#L33), while "Apache License 2.0" is used in [ros2cli's package.xml](https://github.com/ros2/ros2cli/blob/478a430dfb46403d6b156a6b1dc8f531e9a4b3d8/ros2pkg/package.xml#L14)